10/README.md

```diff
@@ -22,14 +22,14 @@ Using random can be hard; it is both too chaotic and sometimes not random enough
 
 Taking a closer look, you can see the [```sin()```](../glossary/?search=sin) wave crest at ```-1.5707``` and ```1.5707```. I bet you now understand why - it's where the maximum and minimum of the sine wave happens.
 
-If look closely at the random distribution, you will note that the there is some concentration around the middle compared to the edges.
+If you look closely at the random distribution, you will note that the there is some concentration around the middle compared to the edges.
 
 <div class="simpleFunction" data="y = rand(x);
 //y = rand(x)*rand(x);
 //y = sqrt(rand(x));
 //y = pow(rand(x),5.);"></div>
 
-A while ago [Pixelero](https://pixelero.wordpress.com) published an [interesting article about random distribution](https://pixelero.wordpress.com/2008/04/24/various-functions-and-various-distributions-with-mathrandom/). I've added some of the functions he uses in the previous graph for you to play with and see how the distribution can be changed. Uncomment the functions and see what happens.
+A while ago [Pixelero](https://pixelero.wordpress.com) published an [interesting article about random distribution](https://pixelero.wordpress.com/2008/04/24/various-functions-and-various-distributions-with-mathrandom/). I've added some of the functions he uses in the previous graph for you to play with to see how the distribution can be changed. Uncomment the functions and see what happens.
```

Comment:
Shorten "to play with to see how the distribution can be changed." as "to play with the distribution."

Reply:
Mmm... not sure. Seams like the 'to' change the meaning